lib/puppet/http/service/report.rb

```diff
@@ -26,15 +24,13 @@ def put_report(name, report, environment:, ssl_context: nil)
 
     @session.process_response(response)
 
-    return response.body.to_s if response.success?
-
-    server_version = response[Puppet::HTTP::HEADER_PUPPET_VERSION]
-    if server_version && SemanticPuppet::Version.parse(server_version).major < MAJOR_VERSION_JSON_DEFAULT &&
```

Comment:
It looks  like `MAJOR_VERSION_JSON_DEFAULT` isn't used anywhere else in this file. Is there a reason you're keeping the definition on line 5?

Reply:
ah no, let me take that out.

Reply:
Updated.